CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# 0.4.1.pre
+
+- Added the `EventSourcing::ActiveRecord::Event#reify` method
+
+# 0.3.1.pre
+
+- Renamed the `EventSourcing::Rails` module to `EventSourcing::ActiveRecord` (@nullobject)
+    - Renamed the `EventSourcing::Rails::CommandRecord` to `EventSourcing::ActiveRecord::Command`
+    - Renamed the `EventSourcing::Rails::EventRecord` to `EventSourcing::ActiveRecord::Event`
```

Comment:
FYI: also creating a Changelog. If this RFC PR gets reject, I'll open a separate one just for the CHANGELOG.